qbittorrent/umbrel-app.yml

```diff
@@ -0,0 +1,30 @@
+manifestVersion: 1
+id: qbittorrent
+category: Networking
+name: qBittorrent
+version: "4.5.2"
+tagline: A fast, easy and free BitTorrent client
```

Comment:
This is the tagline for [Transmission](https://apps.umbrel.com/app/transmission)>

We could use the tagline from https://www.qbittorrent.org/:
`Free and reliable P2P Bittorrent client`